cmd/db_gen_migrations.go

```diff
@@ -0,0 +1,272 @@
+package cmd
```

Comment:
Good idea to put this script, but it requires a good explanation of what it is exactly and what it can be used for.

Reply:
This script cannot be used for the second time, but most of its code likely to be reused.

Reply:
but so, still require a big piece of explanation of what it is :-)

Reply:
fixed